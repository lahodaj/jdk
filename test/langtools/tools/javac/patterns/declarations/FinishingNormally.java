/**
 * @test
 * @enablePreview
 * @compile/fail/ref=FinishingNormally.out -XDrawDiagnostics FinishingNormally.java
 */
public class FinishingNormally {
    public pattern FinishingNormally(String content) {}
}