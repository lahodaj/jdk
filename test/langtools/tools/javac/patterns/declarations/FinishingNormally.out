FinishingNormally.java:7:55: compiler.err.missing.match.stmt
- compiler.note.preview.filename: FinishingNormally.java, DEFAULT
- compiler.note.preview.recompile
1 error